src/Commands/TerminusCommand.php

```diff
@@ -2,34 +2,36 @@
 
 namespace Pantheon\Terminus\Commands;
 
+use Pantheon\Terminus\Config;
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
 use Symfony\Component\Console\Input\InputAwareInterface;
 use Robo\Contract\OutputAwareInterface;
 use Robo\Common\OutputAwareTrait;
 use Robo\Common\InputAwareTrait;
 use Terminus\Models\Auth;
 
-abstract class TerminusCommand implements InputAwareInterface, OutputAwareInterface
+abstract class TerminusCommand implements InputAwareInterface, LoggerAwareInterface, OutputAwareInterface
 {
     use InputAwareTrait;
+    use LoggerAwareTrait;
     use OutputAwareTrait;
 
     /**
-     * @var Config
-     */
-    protected $config;
-    /**
      * @var boolean True if the command requires the user to be logged in
      */
     protected $authorized = false;
+    /**
+     * @var Config
+     */
+    protected $config;
 
     /**
      * TerminusCommand constructor
-     *
-     * @param Config $config Terminus configuration object
      */
-    public function __construct($config)
+    public function __construct()
     {
-        $this->config = $config;
+        $this->config = new Config();
```

Comment:
Injecting this means that I won't be able to let Robo work its inputter/outputter/logger-injecting magic.